cmd/ilm-export.go

```diff
@@ -86,6 +88,10 @@ func mainILMExport(ctx *cli.Context) error {
 
 	ilmCfg, err := client.GetLifecycle()
 	fatalIf(err.Trace(args...), "Unable to get lifecycle configuration")
+	if len(ilmCfg.Rules) == 0 {
```

Comment:
We should allow default config export @BigUstad to allow displaying the data structure, to allow for helping them write the configuration IMO.